### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by RikuVirtanen**
Implemented Task for downloading files through SFTP connection using Frends Tasks guidelines and targeting net6.0, netstandard2.0 and net471. Implemented by referencing Cobalt as baseline.

### CODE REVIEW

File: Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/FileTransferInfo.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
+﻿namespace Frends.SFTP.DownloadFiles.Definitions
+{
+    internal class FileTransferInfo
+    {
+        public string TransferName { get; set; } // 0
```
Comment (jefim) [inline Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/FileTransferInfo.cs:5]: At this point I do not know what the //0 and //1 and //2 mean in this file and I am too afraid to ask :)

File: Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.cs [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+namespace Frends.SFTP.DownloadFiles
+{
+    public class SFTP
+    {
+        private static BatchContext _batchContext;
+
+        /// <summary>
+        /// Downloads file through SFTP connection.
+        /// [Documentation](https://tasks.frends.com/tasks#frends-tasks/Frends.SFTP.DownloadFiles)
```
Comment (jefim) [inline Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.cs:20]: Replace # -> /, so its https://tasks.frends.com/tasks/frends-tasks/Frends.SFTP.DownloadFiles

File: Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.cs [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+            }
+            catch (Exception e)
+            {
+                return $"Error while creating log: \n{e.ToString()}";
+            }
+        }        
+
+        /// <summary>
+        /// Initializes SFTPLogger.
```
Comment (jefim) [inline Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.cs:111]: Is this comment needed?

File: Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/SingleFileTransfer.cs [hunk 4]
```diff
@@ -0,0 +1,503 @@
+﻿using System.Text;
+using Renci.SshNet;
+using Renci.SshNet.Sftp;
+
+namespace Frends.SFTP.DownloadFiles.Definitions
+{
+    /// <summary>
+    /// Class for executing single file transfer.
```
Comment (jefim) [inline Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/SingleFileTransfer.cs:8]: Del?

File: Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/FileTransferInfo.cs (hunk 1 above)
Comment (RikuVirtanen) [reply Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/FileTransferInfo.cs:5]: I have no idea. I copied this from Cobalt. Will delete them.

File: Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.cs (hunk 2 above)
Comment (RikuVirtanen) [reply Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.cs:20]: Fixed

File: Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.cs (hunk 3 above)
Comment (RikuVirtanen) [reply Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.cs:111]: Deleted comment

File: Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/SingleFileTransfer.cs (hunk 4 above)
Comment (RikuVirtanen) [reply Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/SingleFileTransfer.cs:8]: Deleted

**Pull request merged**